File: APIs/MongoDb.API/Models/TickerModel.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace MongoDb.API.Models;

public class TickerModel
{
    [BsonId]
    [BsonRepresentation(BsonType.ObjectId)]
    public ObjectId? Id { get; set; }
    public string Company { get; set; }
    public decimal Value { get; set; }
    
    [BsonDateTimeOptions(Kind = DateTimeKind.Local)]
    public DateTime Date { get; set; }
}

File: Frontend/Angular.WebUI/ClientApp/src/app/mongoDbService/tickerModel.ts
export class TickerModel {
  id: string;
  company: string;
  value: string;
  date: string;

  constructor(id: string, company: string, dateTime: string, value: string) {
    this.id = id;
    this.company = company;
    this.date = dateTime;
    this.value = value;
  }
}
